conjure-java-undertow-runtime/src/test/java/com/palantir/conjure/java/undertow/runtime/OptionsRequestTest.java

```diff
@@ -20,29 +20,22 @@
 
 import com.google.common.collect.Iterables;
 import com.google.common.net.HttpHeaders;
-import com.google.errorprone.annotations.MustBeClosed;
 import com.palantir.conjure.java.undertow.lib.Endpoint;
 import io.undertow.Undertow;
 import io.undertow.server.handlers.ResponseCodeHandler;
 import io.undertow.util.HttpString;
 import io.undertow.util.Methods;
 import java.io.IOException;
+import java.net.HttpURLConnection;
 import java.net.InetSocketAddress;
+import java.net.URL;
 import java.util.UUID;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.Response;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public final class OptionsRequestTest {
 
-    private static final OkHttpClient client = new OkHttpClient.Builder()
-            .followRedirects(false)
-            .retryOnConnectionFailure(false)
```

Comment:
I didnt do anything specific to have this behavior carried forward. I am presuming this is an okhttp client "-ism"